src/main/app/case/from-api-format.ts

```diff
@@ -94,7 +94,7 @@ const fields: FromApiConverters = {
     dateSubmitted: new Date(data.dateSubmitted as string),
   }),
   dueDate: data => ({
-    dueDate: dayjs(data.dueDate).format('MMMM Do YYYY'),
+    dueDate: dayjs(data.dueDate).format('MMMM D YYYY'),
```

Comment:
is this a requirement for all places where the date appears? if not, its risky to change it here.

Reply:
I've checked with Michael, was just pushing it for ease while waiting for a response

Reply:
@velimd the GOVUK style guide that Michael sent me indicates that the ordinal numbers shouldn't be used (i.e keep this change ^)
https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style